reactos/drivers/usb/usbhub_new/usbhub.c

```diff
@@ -2611,7 +2611,8 @@ USBD_GetDeviceInformationEx(IN PUSBHUB_PORT_PDO_EXTENSION PortExtension,
 
         if (!NodeInfo)
         {
-            Status = STATUS_INSUFFICIENT_RESOURCES;
+            ExFreePoolWithTag(DeviceInfo, USB_HUB_TAG);
+            return STATUS_INSUFFICIENT_RESOURCES;
```

Comment:
usbhub.c:2615: Now that !NodeInfo goes to exit, Status will always be successful down in line 2620, so you can remove that check.